apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/ApplicationBuilder.kt

```diff
@@ -80,7 +80,12 @@ class ApplicationBuilder {
         )
     }
 
-    private val pdfGenerator = PdfGeneratorKlient(httpClient(), env.requireEnvValue("ETTERLATTE_PDFGEN_URL"))
+    private val brevbaker =
+        BrevbakerKlient(
+            httpClient("BREVBAKER_SCOPE"),
```

Comment:
Skal ikke denne være `httpClient(env.requireEnvValue("BREVBAKER_SCOPE")),`

Reply:
httpClient kjører `get(scope)` så den henter ut verdien for oss